tests/async/models.py

```diff
@@ -1,5 +1,9 @@
 from django.db import models
 
 
+class RelatedModel(models.Model):
+    simple = models.ForeignKey("SimpleModel", models.CASCADE, null=True)
+
+
```

Comment:
this model doesn't need to be in this commit now

```suggestion
```

Reply:
Thanks, missed it somehow. 😢